### PR TITLE
Removed the word root and added info about local

### PR DESCRIPTION
**Pull request by joseph619**
Removed the word root (which caused confusion among readers), and added info that local has to be created manually first.

### CODE REVIEW

File: src/help/guides/reading-local-manga.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ lang: en-US
 
 # Reading local manga
 
-You have to place your manga inside the folder `Tachiyomi/local` located in the root of your internal storage or external SD card, usually `/sdcard` and `/storage/18F5-2C11` (with different values).
+You have to first create a folder named `local` in Tachiyomi folder, which is located in the phone's `Internal storage` or external SD card. Then place your manga inside the folder `Tachiyomi/local`, usually `/sdcard` and `/storage/18F5-2C11` (with different values).
```
Comment (arkon) [inline src/help/guides/reading-local-manga.md]: Did you mean to keep the ending?

Comment (joseph619) [reply src/help/guides/reading-local-manga.md]: Thanks for replying, what do you mean by 'the ending' here?

Comment (arkon) [reply src/help/guides/reading-local-manga.md]: The ", usually `/sdcard` and `/storage/18F5-2C11` (with different values)" part. It sounded like you were trying to reword it to be less confusing, but now it just doesn't make sense.

**Comment by joseph619** [comment]
Oh, that part wasn't written by me. I had kept that part untouched, should I rewrite it as well?

**Comment by arkon** [comment]
Yes, the sentence doesn't actually flow in its current state.

File: src/help/guides/reading-local-manga.md (hunk 1 above)
Comment (joseph619) [reply src/help/guides/reading-local-manga.md]: Oh, that part wasn't written by me. I left it untouched, should I rewrite that as well?

**Comment by joseph619** [comment]
Ok I will edit that too
